capitals.rb

```diff
@@ -1,153 +1,204 @@
+                                                                                        }, {
+                                                                                          name: "Utah",
+                                                                                          capital: "Salt Lake City"
+                                                                                          }, {
+                                                                                            name: "Vermont",
+                                                                                            capital: "Montpelier"
+                                                                                            }, {
+                                                                                              name: "Virginia",
+                                                                                              capital: "Richmond"
+                                                                                              }, {
+                                                                                                name: "Washington",
+                                                                                                capital: "Olympia"
+                                                                                                }, {
+                                                                                                  name: "West Virginia",
+                                                                                                  capital: "Charleston"
+                                                                                                  }, {
+                                                                                                    name: "Wisconsin",
+                                                                                                    capital: "Madison"
+                                                                                                    }, {
+                                                                                                      name: "Wyoming",
+                                                                                                      capital: "Cheyenne"
+                                                                                                      }]
+
+
+# an array of state hashes
+
+
+# states = states.shuffle
+
+puts "Hello do you know your states and capitals? Just note your score will be
+counted based on every correct answer. You will also lose points for every incorrect
+response. Please use CAPITALIZATION! "
+
+states = states.shuffle
+score_var = 0
+
+#
+states.each do |state|
+  state[:num_right] = 0
+  state[:num_wrong] = 0
+end
+
+
+
+# i = 0
+# while i < 5 do
+#    puts i
+#    i += 1
+# end
+
+  #
+    while true
+      #Ask the user to name the capital of each state.
```

Comment:
this loop never ends! but you probably know this. need some kind of `break` statement inside of a conditional.